provider/grpc.go

```diff
@@ -139,6 +139,41 @@ func (g *GRPCClient) GetOpenOrders(ctx context.Context, market string, owner str
 	return g.apiClient.GetOpenOrders(ctx, &pb.GetOpenOrdersRequest{Market: market, Address: owner, OpenOrdersAddress: openOrdersAddress, Project: project})
 }
 
+// GetOpenPerpOrders returns all opened perp orders
+func (g *GRPCClient) GetOpenPerpOrders(ctx context.Context, owner string, accountAddress string, contracts []common.PerpContract, project pb.Project) (*pb.GetOpenPerpOrdersResponse, error) {
```

Comment:
can you make this all consistent? Either make it all `request *pb.type` or enumerate the arguments in the function signature. Let's not try to avoid mixing it in new code